asistente_ladm_col/asistente_ladm_col_plugin.py

```diff
@@ -1032,20 +1032,30 @@ def call_fill_topology_tables_morebfs_less(self, *args):
     def call_fill_right_of_way_relations(self, *args):
         self.right_of_way.fill_right_of_way_relations(self.get_db_connection())
 
+    @_validate_if_plot_is_selected
     @_validate_if_wizard_is_open
     @_qgis_model_baker_required
     @_db_connection_required
     @_cadastral_cartography_model_required
     @_survey_model_required
     def call_ant_map_report_generation(self, *args):
-        self.report_generator.generate_report(self.get_db_connection(), ANT_MAP_REPORT)
+        ant_map_report = ANTMapReport(self.get_db_connection())
+        ant_map_report.enable_action_requested.connect(self.enable_action)
+        ant_map_report.exec_()
+        ant_map_report.enable_action_requested.disconnect(self.enable_action)
```

Comment:
This is redundant in Qt. If you'll destroy the object, all its connections will be gone anyways.

Reply:
That's right, thanks for the observation. I will keep it in mind for future use.